regulations/views/comment.py

```diff
@@ -100,7 +100,7 @@ def submit_comment(request):
         logger.error(message)
         return JsonResponse({'message': message}, status=403)
 
-    files = extract_files(body)
+    files = extract_files(body['general_comment'])
```

Comment:
This doesn't seem to match the front end's expectations: https://github.com/eregs/regulations-site/blob/master/regulations/static/regulations/js/source/views/comment/comment-review-view.js#L108 + https://github.com/eregs/regulations-site/blob/master/regulations/static/regulations/js/source/views/comment/comment-review-view.js#L76